config/doctrine/Assignment.orm.xml

```diff
@@ -40,13 +41,18 @@
                 <join-column name="user_id" referenced-column-name="id"/>
```

Comment:
I know that it's not related to your work but could you please add `nullable="false"` here? Thanks